### PR TITLE
Subs 552 new repo

### PR DESCRIPTION
**Pull request by neilgoodgame**
Extracted the subs-ena-data-model from the subs project

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,18 +1,94 @@
...
+        classpath 'org.hidetake:gradle-ssh-plugin:1.1.3'
+    }
+}
+
+version '1.0.3-SNAPSHOT'
+
 apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'maven'
+apply plugin: 'eclipse'
```
Comment (ke4) [inline build.gradle]: Do we need this 'eclipse' plugin?
Here is the docs: https://docs.gradle.org/3.3/userguide/eclipse_plugin.html
If not, then better to remove it.

**Review by ke4 — APPROVED** [review]
We can remove that plugin later from the build, if we do not need it.

**Review by ke4 — APPROVED** [review]

**Comment by ke4** [comment]
I can't merge it. Probably some permission problem. :(

File: build.gradle (hunk 1 above)
Comment (neilgoodgame) [reply build.gradle]: removed it

**Pull request merged**